### PR TITLE
backend/drm: make page-flip error scarier

### PR DESCRIPTION
**Pull request by emersion**
The logs don't currently display the importance of a line. It's easy to
read "skipping page-flip" as a debug message instead of an error
message.

Change the error message to make it more explicit.

References: https://github.com/swaywm/wlroots/pull/2147

### CODE REVIEW

**Comment by kennylevinsen** [comment]
Much better! :)

**Comment by emersion** [comment]
Note, the Arch build fails because of:

```
> clang
clang: error while loading shared libraries: libicuuc.so.67: cannot open shared object file: No such file or directory
```

**Comment by ddevault** [comment]
Has that been reported to Arch upstream?

**Comment by ddevault** [comment]
Thanks!

**Comment by emersion** [comment]
Still investigating. On my system I have icu 67.1-1, on builds.sr.ht we have 65.1-3. Both are up-to-date, clang works locally. clang depends on icu through libxml2. Both my local machine and builds.sr.ht have libxml2 2.9.10-2.

tl;dr probably an issue due to a partial update on one of Arch package mirrors.

**Comment by ddevault** [comment]
...so, report your findings upstream?

**Comment by emersion** [comment]
Yeah, definitely. Still collecting more info for now.

**Comment by ddevault** [comment]
Thanks :)
